x-pack/plugins/event_log/README.md

```diff
@@ -125,9 +125,9 @@ Here's the event written to the event log index:
       "duration": 1000000
     },
     "kibana": {
-      "namespace": "default",
       "saved_objects": [
         {
+          "namespace": "default",
```

Comment:
The `namespace` property shouldn't exist here, for the default space (since it's `undefined`).  Might want to make a note in the README about this, as the property will exist for spaces besides the default space.